share/spice/keybase/keybase.js

```diff
@@ -44,15 +44,25 @@
                 sourceUrl: 'https://keybase.io/' + user.basics.username
             },
             normalize: function(item) {
+                var userAccounts = [];
+                userAccounts.push({
+                        text: keybase_key_fingerprint(item.public_keys.primary.key_fingerprint),
+                        href: 'https://keybase.io/' + item.basics.username + '/key.asc'});
+                var accountGroup = item.proofs_summary.all;
+                var accountName;
+                for( accountName in accountGroup ) {
```

Comment:
I would opt for `$.each()` which is a little safer because it ensures `hasOwnProiperty` on each of the keys as it iterates over the object.